core/src/main/java/hudson/lifecycle/Lifecycle.java

```diff
@@ -304,7 +304,7 @@ public void onStop(@NonNull String user, @CheckForNull String remoteAddr) {
      * @param timeout The amount by which to extend the timeout.
      * @param unit The time unit of the timeout argument.
      *
-     * @since TODO
+     * @since 2.335
```

Comment:
This one is wrong. Line was added (badly) in https://github.com/MarkEWaite/jenkins/commit/7915d822cb84a6a17568dfdc55cfdb21c49dc97f.

Reply:
Apologies if I made a mistake there, but I'm not sure of the mistake I made.  Can you clarify for me so that I can correct it?

The API `onExtendTimeout(long timeout, @NonNull TimeUnit unit)` was added in commit 3866e93a364 as far as I can tell.  The first release tag to include that was [2.335](https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.335) as part of jenkinsci/jenkins#6237.

Reply:
Ah, I missed that these weren't released earlier, just saw that a different PR was identified.